### PR TITLE
Key utils

### PR DESCRIPTION
**Pull request by festinalente91**
updated key utils

### CODE REVIEW

File: src/server/key-utils.js [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+export const KeyUtils = {
+  getReversedTime: (time = new Date()) => {
+    return MAX_TIME - Number(new Date(time));
+  },
+  genTimeHash: (time) => {
+    const reversedTime = KeyUtils.getReversedTime(time);
+    const uuidKey = uuid();
+    return `${reversedTime}-${uuidKey}`;
+  },
+  getIdxKey: (entity, timeHash, ...options) => {
```
Comment (hewonjeong) [inline src/server/key-utils.js:61]: Elegant!

File: src/server/key-utils.js [hunk 2]
```diff
@@ -0,0 +1,101 @@
...
+    // in options, larger notion should came first
+    // ex) item, image
+    let key = `${entity}-${STATE.ALIVE}`;
+    options.map((option) => {
+      key = `${key}-${option}`;
+    });
+    return `${key}-${timeHash}`;
+  },
+  getKeysByArea: (lat, lng, precision) => {
```
Comment (hewonjeong) [inline src/server/key-utils.js:70]: How about rename this function?(e.g. getNeighbors or getNeighborsByArea)
Cause 'Keys' sounds like our item or image key('item-{state}-{geohash}-~~~').

Comment (festinalente91) [reply src/server/key-utils.js:70]: Right. it would be better.

File: src/server/items/controllers.js [hunk 3]
```diff
@@ -103,14 +103,21 @@ export default {
         return cb(new APIError(getErr));
       }
       const item = value;
-      const itemTimeStamp = new Timestamp(item.createdDate).getTimestamp();
+      const TimeStamp = new Timestamp(item.createdDate).getTimestamp();
```
Comment (ovekyc) [inline src/server/items/controllers.js:106]: Please stick to use 'Camel Case' naming convention
https://github.com/airbnb/javascript#naming-conventions

**Comment by ovekyc** [comment]
LGTM

**Comment by hewonjeong** [comment]
LGTM

**Comment by hewonjeong** [comment]
Lgtm again

**Pull request merged**